packages/quantic/force-app/main/default/lwc/quanticResultTemplate/quanticResultTemplate.js

```diff
@@ -30,6 +30,13 @@ export default class QuanticResultTemplate extends LightningElement {
    */
   @api resultPreviewShouldNotBeAccessible = false;
   
+  /**
+   * Indicates whether the bottom border of the result template should be hidden..
+   * @api
+   * @type {boolean}
+   */
+  @api hideBottomBorder = false;
```

Comment:
This property is tightly coupled with the child result templates. Later on, the component styling might change in a way that makes removing the border unnecessary. Setting this property would then break the component styling.

Instead of exposing a property like this, what would you think of inferring the value by checking if a component is configured in the `children` slot?

Reply:
yeah I agree, wasn't even confident about this property, not sure tho how we can detect if a template is configured in the `children` slot.

One solution I thought about is avoid adding this `hideBottomBorder` property and completely remove the border bottom of the QuanticResultTemplate component.
Then if we want to have a border bottom we would only need to do the following:
```
<div class="slds-border-bottom">
   <c-quantic-result-template>....</c-quantic-result-template>
</div>
```
But I think this solution also  falls in the breaking change category.

Reply:
A named slot elements can be accessed like this (see [LWC doc](https://lwc.dev/guide/composition#access-elements-passed-via-slots)):

```javascript
const childTemplates = this.querySelectorAll('*[slot="children"]');
```

In your case, you could have something like this:
```javascript
get hasChildTemplates() {
  return this.querySelectorAll('*[slot="children"]').length > 0;
  // or use `querySelector` to avoid retrieving all elements
  return !!this.querySelector('*[slot="children"]');
}
```